babelon/hp-it.babelon.tsv

```diff
@@ -1,2900 +1,3489 @@
 source_language	source_value	subject_id	predicate_id	translation_language	translation_value	translation_status	translator	translator_expertise	comment	translation_date
-en	Hydrocephalus	HP:0000238	rdfs:label	it	Idrocefalo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Severe global developmental delay	HP:0011344	rdfs:label	it	Grave ritardo globale dello sviluppo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Retinal hemorrhage	HP:0000573	rdfs:label	it	Emorragia retinica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Intracranial hemorrhage	HP:0002170	rdfs:label	it	Emorragia intracranica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Delayed speech and language development	HP:0000750	rdfs:label	it	Ritardo nello sviluppo del linguaggio e della parola	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Bulbous nose	HP:0000414	rdfs:label	it	Naso bulboso	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Hypotonia	HP:0001252	rdfs:label	it	Ipotonia	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Multicystic kidney dysplasia	HP:0000003	rdfs:label	it	Displasia renale multicistica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
```

Comment:
What does "preview" mean in the comment?

Reply:
I don't know, this is not a new line, it comes from the previously existing file...  see
https://github.com/obophenotype/hpo-translations/blob/c14c23fc4e4acba7b47db15b447d919faef03a10/babelon/hp-it.babelon.tsv#L1010

Reply:
Peter must have added it. Ok, not important, I don't think! leave it as is now!